src/Microsoft.Identity.Client/Cache/NoLockTokenCacheProxy.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.Cache
+{
+    internal class NoLockTokenCacheProxy : ITokenCache
+    {
+        private readonly TokenCache _tokenCache;
+
+        internal NoLockTokenCacheProxy(TokenCache tokenCache)
+        {
+            _tokenCache = tokenCache;
+
+        }
+
+        public void Deserialize(byte[] msalV2State) => throw new NotImplementedException();
+
+        public void DeserializeAdalV3(byte[] adalV3State)
+        {
+#if !ANDROID_BUILDTIME && !iOS_BUILDTIME
```

Comment:
BUILDTIME flags are used to hide APIs. That call will occur at runtime and result in a MethodNotFoundException. Consider using !ANDROID && !iOS instead